### PR TITLE
Fix hva stop job

### PR DESCRIPTION
**Pull request by hansode**
Detail #596

### CODE REVIEW

**Comment by hansode** [comment]
```
01:10:21 D, [2015-06-01T01:08:19.630680 #1894] DEBUG -- Lxc: Session ID: bf2666fd72850986951c86924acdb451ba4b1715: Command Result: fail (exit code=1)
01:10:21 Command PID: 28617
01:10:21 ##STDERR=>
01:10:21 Container already exists
01:10:21 E, [2015-06-01T01:08:19.631224 #1894] ERROR -- Lxc: Session ID: bf2666fd72850986951c86924acdb451ba4b1715: Dcmgr::Helpers::CliHelper::ShellRunner::CommandError Unexpected exit code=1: lxc-create -f /var/lib/wakame-vdc/instances/i-e2iye4ti/lxc.conf -n i-e2iye4ti
01:10:21 Command PID: 28617
01:10:21 ##STDERR=>
01:10:21 Container already exists from /opt/axsh/wakame-vdc/dcmgr/lib/dcmgr/helpers/cli_helper.rb:191:in `block in run!'
01:10:21 2015-06-01 01:08:19 JobContext thr=JobWorker[0/1] [ERROR]: Job failed bf2666fd72850986951c86924acdb451ba4b1715 [ run_local_store ]: Unexpected exit code=1: lxc-create -f /var/lib/wakame-vdc/instances/i-e2iye4ti/lxc.conf -n i-e2iye4ti
01:10:21 Command PID: 28617
01:10:21 ##STDERR=>
01:10:21 Container already exists
01:10:21 2015-06-01 01:08:19 JobContext thr=JobWorker[0/1] [ERROR]: Caught Dcmgr::Helpers::CliHelper::ShellRunner::CommandError: Unexpected exit code=1: lxc-create -f /var/lib/wakame-vdc/instances/i-e2iye4ti/lxc.conf -n i-e2iye4ti
01:10:21 Command PID: 28617
01:10:21 ##STDERR=>
01:10:21 Container already exists
01:10:21 	/opt/axsh/wakame-vdc/dcmgr/lib/dcmgr/helpers/cli_helper.rb:191:in `block in run!'
01:10:21 	/opt/axsh/wakame-vdc/dcmgr/lib/dcmgr/helpers/cli_helper.rb:189:in `tap'
01:10:21 	/opt/axsh/wakame-vdc/dcmgr/lib/dcmgr/helpers/cli_helper.rb:189:in `run!'
01:10:21 	/opt/axsh/wakame-vdc/dcmgr/lib/dcmgr/helpers/cli_helper.rb:106:in `sh'
01:10:21 	/opt/axsh/wakame-vdc/dcmgr/lib/dcmgr/drivers/hypervisor/linux_hypervisor/linux_container/lxc.rb:65:in `run_instance'
01:10:21 	/opt/axsh/wakame-vdc/dcmgr/lib/dcmgr/task.rb:178:in `call'
01:10:21 	/opt/axsh/wakame-vdc/dcmgr/lib/dcmgr/task.rb:178:in `invoke!'
01:10:21 	/opt/axsh/wakame-vdc/dcmgr/lib/dcmgr/task.rb:249:in `invoke'
01:10:21 	/opt/axsh/wakame-vdc/dcmgr/lib/dcmgr/rpc/hva_handler.rb:327:in `block in <class:HvaHandler>'
01:10:21 	/opt/axsh/wakame-vdc/dcmgr/vendor/bundle/ruby/2.0.0/gems/isono-0.2.20/lib/isono/runner/rpc_server.rb:69:in `instance_eval'
01:10:21 	/opt/axsh/wakame-vdc/dcmgr/vendor/bundle/ruby/2.0.0/gems/isono-0.2.20/lib/isono/runner/rpc_server.rb:69:in `block in job'
01:10:21 	/opt/axsh/wakame-vdc/dcmgr/vendor/bundle/ruby/2.0.0/gems/isono-0.2.20/lib/isono/rack/proc.rb:25:in `instance_eval'
01:10:21 	/opt/axsh/wakame-vdc/dcmgr/vendor/bundle/ruby/2.0.0/gems/isono-0.2.20/lib/isono/rack/proc.rb:25:in `call'
01:10:21 	/opt/axsh/wakame-vdc/dcmgr/vendor/bundle/ruby/2.0.0/gems/isono-0.2.20/lib/isono/rack/map.rb:52:in `call'
01:10:21 	/opt/axsh/wakame-vdc/dcmgr/vendor/bundle/ruby/2.0.0/gems/isono-0.2.20/lib/isono/rack/map.rb:52:in `call'
01:10:21 	/opt/axsh/wakame-vdc/dcmgr/vendor/bundle/ruby/2.0.0/gems/isono-0.2.20/lib/isono/rack/builder.rb:36:in `call'
01:10:21 	/opt/axsh/wakame-vdc/dcmgr/vendor/bundle/ruby/2.0.0/gems/isono-0.2.20/lib/isono/rack/job.rb:59:in `block (2 levels) in call'
01:10:21 	/opt/axsh/wakame-vdc/dcmgr/vendor/bundle/ruby/2.0.0/gems/isono-0.2.20/lib/isono/node_modules/job_worker.rb:67:in `call'
01:10:21 	/opt/axsh/wakame-vdc/dcmgr/vendor/bundle/ruby/2.0.0/gems/isono-0.2.20/lib/isono/node_modules/job_worker.rb:67:in `block in start'
01:10:21 	/opt/axsh/wakame-vdc/dcmgr/vendor/bundle/ruby/2.0.0/gems/isono-0.2.20/lib/isono/thread_pool.rb:32:in `call'
01:10:21 	/opt/axsh/wakame-vdc/dcmgr/vendor/bundle/ruby/2.0.0/gems/isono-0.2.20/lib/isono/thread_pool.rb:32:in `block (2 levels) in initialize'
```

File: dcmgr/lib/dcmgr/rpc/hva_handler.rb [hunk 1]
```diff
@@ -437,14 +437,14 @@ def wait_volumes_available
...
-          update_instance_state({:state=>:stopping})
-          ignore_error { terminate_instance(false) }
-        ensure
-          #
-          update_instance_state_to_terminated({:state=>:stopped, :host_node_id=>nil})
-        end
+        @hva_ctx.logger.info("Turning power off")
+        task_session.invoke(@hva_ctx.hypervisor_driver_class,
+                            :terminate_instance, [@hva_ctx])
+        update_instance_state({:state=>:stopped}, ['hva/instance_turnedoff'])
```
Comment (unakatsuo) [inline dcmgr/lib/dcmgr/rpc/hva_handler.rb]: ``:host_node_id=>nil`` also needs to be set.

**Comment by Metallion** [comment]
As @unakatsuo explained in https://github.com/axsh/wakame-vdc/issues/596, `stop` is currently meant to delete the local volume. It should not have the same functionality as `poweroff`. I think we should re-evaluate whether or not `start` and `stop` are really needed.

**Comment by hansode** [comment]
#603

**Comment by hansode** [comment]
@unakatsuo and me discussed and decided 'API: Retire instance "stop" & "start" operation. #603'.
So this PR is unnecessary.

**Comment by hansode** [comment]
@Metallion please close this PR.

**Pull request closed without merging**